src/arguments.js

```diff
@@ -94,6 +97,8 @@ async function getEventArguments(event) {
         event['note'] = DEFAULT_EMAIL_NOTE;
     if (event.multitenancy === undefined)
         event['multitenancy'] = DEFAULT_MULTI_TENANCY;
+    if (event.selfsignedcerts === undefined)
+        event['selfsignedcerts'] = DEFAULT_SELF_SIGNED_CERTIFICATES;
```

Comment:
Could you try this merge strategy :

```
const defaults = {
  auth: DEFAULT_AUTH,
  tenant: DEFAULT_TENANT,
  ... (etc)
}

Object.assign(defaults, ...event)
```

I think would both clarify the defaults, and make the fallback assignments a one-liner.